test/metadata_closure_test.js

```diff
@@ -4,15 +4,15 @@ describe('A test', {
   metadata: { a: 1 },
 
   tests: function() {
-    it('should be able to access its own metadata, given by the suite', function(_metadata) {
-      expect(_metadata).to.eql({ a: 1 });
+    it('should be able to access its own metadata, given by the suite', function() {
+      expect(this.test.metadata).to.eql({ a: 1 });
```

Comment:
do you think it's preferable to have `this.metadata` or `this.test.metadata`? To me the former seems like it makes more sense, since you're calling it in the context of the test itself.